core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/model/instancegroup/template/InstanceTemplateV1Base.java

```diff
@@ -22,6 +22,9 @@
     @ApiModelProperty(TemplateModelDescription.AZURE_PARAMETERS)
     private AzureInstanceTemplateV1Parameters azure;
 
+    @ApiModelProperty(TemplateModelDescription.AZURE_PARAMETERS)
```

Comment:
Not a big deal but the description is incorrect (copy-paste I suppose).

Reply:
can you please modify @topolyai5 ?